doc/help.md

```diff
@@ -889,6 +889,86 @@ JOB:
 
 ### External
 
+<h4 id="[">[</h4>
+
+    [ EXPR+ ]
+
+This is an alias for the `test` builtin. Unlike `test`, expressions must be
+followed by a closing ']'.
+
+Tips:
+
+Don't forget to leave a space between the brackets and the expression. "[" is
+a command, not syntax, so the shell will fail to recognize it without the
+spaces.
+
+
+<h4 id="test">test</h4>
+
+    test EXPR+
+
+Evaluates a conditional expression and returns 0 (true) or 1 (false) as result.
+The number of elments in the expression depends on the type of test.
+
+File tests:
+
+    -a FILE  True if FILE exists (recommended to use -e instead).
+    -b FILE  True if FILE is a block special file.
+    -c FILE  True if FILE is a character special file.
+    -d FILE  True if FILE is a directory.
+    -e FILE  True if FILE exists.
+    -f FILE  True if FILE is a regular file.
+    -g FILE  True if FILE has the sgid bit set.
+    -G FILE  True if current user's group is also FILE's group.
+    -h FILE  True if FILE is a symbolic link.
+    -L FILE  True if FILE is a symbolic link.
+    -k FILE  True if FILE has the sticky bit set.
+<!--    -N FILE  True if FILE was modified since last read (mtime newer than atime).-->
```

Comment:
The comments in the middle cause some formatting problems in HTML.  Try building the HTML and seeing how it looks (and let me know if you can't)

BTW I found some smart unicode quotes in some files.  That could be related to the Unicode errors

Reply:
You are right, I didn't notice it. I have moved it at the end of the block, it doesn't affect the HTML output there.